src/components/DocPages/Container.jsx

```diff
@@ -1,9 +1,11 @@
 import styled from "styled-components"
+import * as styleVars from "../variable"
 
 export const Container = styled.section`
   display: flex;
   max-width: 100%;
   align-items: center;
+  background: linear-gradient(${styleVars.backgroundGradientStart}, ${styleVars.backgroundGradientEnd});
```

Comment:
nit: use specific color (in this case lightSecondaryBlue and darkSecondaryBlue) instead of overfitting constants to specific use case

Also let's use a white background for legal text

Reply:
Ok will make the change and forward the commits later today.